django_mri/analysis/interfaces/dmriprep_bak/dmriprep.py

```diff
@@ -0,0 +1,297 @@
+"""
+Definition of the :class:`DmriPrep` interface.
+"""
```

Comment:
Let's not forget to remove the `_bak` module.

Reply:
Sure, once we're confident that the new interface is stable.